bpmnconverter/converter/converter.py

```diff
@@ -20,9 +20,14 @@
         print('No data was parsed, something went wrong')
     else:
 
-        print(json.dumps(bpmn_object))
+        #print(json.dumps(bpmn_object))
```

Comment:
Any reason to leave these commented print statements in?

Reply:
Removed.